### PR TITLE
Is 72 list business form

### PR DESCRIPTION
**Pull request by ctperry0301**

### CODE REVIEW

**Comment by ctperry0301** [comment]
TODO: Link form with Post request on completion

**Review by jpoist97 — COMMENTED** [review]
Look good so far, only thing would be maybe we can split this component across multiple files to make this file a bit smaller. Just an idea though and it works so we can always come back and refactor it later. 

Off the top of my head if you wanted to do it we could probably split the different sections after the conditional statements into different components, but I don't know if it's worth it tbh. Good work though this file is fat

File: frontend/isoko/src/pages/ListBusiness.tsx [hunk 1]
```diff
@@ -192,7 +190,8 @@ const ListBusiness: React.FC = () => {
    const [state, setState] = useState('');
    const [zip, setZip] = useState('');
    const [tags, setTags] = useState([]);
-   const [keywords, setKeywords] = useState('');
+   const [keywords] = useState([]);
```
Comment (jpoist97) [inline frontend/isoko/src/pages/ListBusiness.tsx:193]: We can take this out now right?

File: frontend/isoko/src/pages/ListBusiness.tsx [hunk 2]
```diff
@@ -297,7 +296,8 @@ const ListBusiness: React.FC = () => {
          zip,
          type: isBrickAndMortar === 'true' ? 'B&M' : 'Online',
          tags: tags,
-         keywords: [keywords],
+         category,
+         keywords,
```
Comment (jpoist97) [inline frontend/isoko/src/pages/ListBusiness.tsx:300]: Here as well, looks like this is the only place keywords is used in this file

**Review by jpoist97 — COMMENTED** [review]
Looks pretty good to me, just the one comment about keywords

**Comment by ctperry0301** [comment]
@jpoist97 Good catch, should be ready to go now

**Review by jpoist97 — APPROVED** [review]

**Pull request merged**